bitbots_bringup/launch/load_robot_description.launch

```diff
@@ -20,8 +22,13 @@
 
 
     <group if="$(arg wolfgang)">
-        <param name="robot_description" command="$(find xacro)/xacro $(find wolfgang_description)/urdf/wolfgang.urdf.xacro use_fake_walk:=$(arg use_fake_walk) sim_ns:=$(arg sim_ns) --inorder" />                
-        <param name="urdf" value="$(find wolfgang_description)/urdf/wolfgang.urdf.xacro" />
+        <param name="robot_description" command="$(find xacro)/xacro $(find wolfgang_description)/urdf/wolfgang_basler.urdf.xacro use_fake_walk:=$(arg use_fake_walk) sim_ns:=$(arg sim_ns) --inorder" />                
```

Comment:
Is it correct that the basler urdf is always used for the robot description?

Reply:
Oh, I oversaw this parameter.